modules/ROOT/pages/editor-model.adoc

```diff
@@ -0,0 +1,10 @@
+= TinyMCE Model options
+
+:title_nav: Models
```

Comment:
This is technically a useless attribute atm and I'd like to remove it, but for now I guess it should match the actual name you've used in the nav.
```suggestion
:title_nav: Editor content models
```